src/screens/points/content/PointsContent.tsx

```diff
@@ -0,0 +1,796 @@
+        background="accent"
+        height={{ custom: height }}
+        width={{ custom: width }}
+        borderRadius={18}
+        style={{ overflow: 'hidden' }}
+      >
+        <ShimmerAnimation
+          color={colors.alpha(colors.blueGreyDark, 0.06)}
+          width={width}
+          // @ts-ignore
+          gradientColor={colors.alpha(colors.blueGreyDark, 0.06)}
+        />
+      </Box>
+    </AccentColorProvider>
+  );
+};
+
+const InfoCard = ({
+  onPress,
+  title,
+  subtitle,
+  mainText,
+  icon,
+  accentColor,
+}: {
+  onPress: () => void;
+  title: string;
+  subtitle: string;
+  mainText: string;
+  icon: string;
+  accentColor: string;
+}) => (
+  <ButtonPressAnimation onPress={onPress} overflowMargin={50}>
+    <Box
+      padding="20px"
+      background="surfaceSecondaryElevated"
+      shadow="12px"
+      height={{ custom: 98 }}
+      borderRadius={18}
+    >
+      <Stack space="12px">
+        <Inline space="4px" alignVertical="center">
+          <Text color="labelSecondary" weight="bold" size="15pt">
+            {title}
+          </Text>
+          <Text color="labelQuaternary" weight="heavy" size="13pt">
+            􀅵
+          </Text>
+        </Inline>
+        <Text color="label" weight="heavy" size="22pt">
+          {mainText}
+        </Text>
+        <Inline space="4px">
+          <Text
+            align="center"
+            weight="heavy"
+            size="12pt"
+            color={{ custom: accentColor }}
+          >
+            {icon}
+          </Text>
+          <Text weight="heavy" size="13pt" color={{ custom: accentColor }}>
+            {subtitle}
+          </Text>
+        </Inline>
+      </Stack>
+    </Box>
+  </ButtonPressAnimation>
+);
+
+export default function PointsContent() {
+  const { colors } = useTheme();
+  const { width: deviceWidth } = useDimensions();
+  const { accountAddress, accountENS } = useAccountProfile();
+  const { setClipboard } = useClipboard();
+  const { data, isFetching, dataUpdatedAt } = usePoints({
+    walletAddress: accountAddress,
+  });
+  const { navigate } = useNavigation();
+
+  const labelSecondary = useForegroundColor('labelSecondary');
+  const pink = useForegroundColor('pink');
+  const yellow = useForegroundColor('yellow');
+
+  const [isToastActive, setToastActive] = useRecoilState(
+    addressCopiedToastAtom
+  );
+
+  useFocusEffect(
+    useCallback(() => {
+      const backAction = () => {
+        navigate(Routes.WALLET_SCREEN);
+        return true;
+      };
+
+      const backHandler = BackHandler.addEventListener(
+        'hardwareBackPress',
+        backAction
+      );
+
+      return () => backHandler.remove();
+    }, [navigate])
+  );
+
+  const onPressCopy = React.useCallback(
+    (onNewEmoji: () => void) => {
+      if (!isToastActive) {
+        setToastActive(true);
+        setTimeout(() => {
+          setToastActive(false);
+        }, 2000);
+      }
+      onNewEmoji();
+      setClipboard(accountAddress);
```

Comment:
shouldn't this be referral code? also curious id this maybe should copy the full link vs just the code

Reply:
fixed this in follow up pr that i'm working on